### PR TITLE
Will Toon first map post

### PR DESCRIPTION
**Pull request by WillToon**
Uploaded a portfolio post for the first map. Unfortunately, I ran into some issues exporting the geometry for CU (and other locations) from OpenStreetMap. If you guys found any specific ways to enter your points of interest to do this, please let me know as I currently just have an unexciting map with a point!

### CODE REVIEW

File: posts/will-toon/cu.html
Comment (brglea) [inline posts/will-toon/cu.html:1]: @WillToon I pulled you raw data and do in fact see what you are talking about, it's just a point. There's something wrong and it's hard for me to point out without seeing your codespace. I also had struggles when trying to accurately pull a polygon, and just one polygon not multiple haha. The first thing I did is actually go to the open street maps website and search for what I was looking for. In your case [CU Boulder campus](https://www.openstreetmap.org/search?query=University%20of%20Colorado%20Boulder%2C%20Colorado%2C%20United%20States#map=16/39.99648/-105.24879) There's a ton of results, so you will need to choose the one that is what you want and try to adapt your search parameters to match the criteria that you see in the table, for example, according to OSM it labels the amenity type as a 'university' not a college like the UTTC example, so pay attention to what the amentiy type name is. I actuality ended up using a a different criteria (building=mosque) that was hyperlinked and that pulled the polygon I wanted, it was a lot of trial and error until it found what I wanted. In your codespace it might look something like:

# Search for University of Colorado Boulder
ucb_gdf = osm.features_from_address(
    'University of Colorado Boulder, Boulder, CO, United States',
    {'amenity': ['university']})
ucb_gdf

However that being said, I had to play around with the spelling, leaving out the city, changing the spelling of the country, accent marks, no accents, etc. of search until the OSM data I wanted correctly pulled, I knew i got the one I wanted according to the OSM id # which a possible one for you is 46226108 and I made sure I saw whatever the id number was before I moved on to the next step.

I think the problem with the background not showing is maybe related to the creating an interactive map step/coding, but I can't tell looking at the code here. I'm hoping if you find the right OSM vector data that will pull a polygon instead of a point. 

An idea, I'm no expert though, make sure you have the code line   geo=True, tiles='EsriImagery',   from the Create an Interactive map section in the textbook because the imagery has to be there for the main map background to show is my thought. But maybe something else is wrong that I'm not aware of.

Anyways, you do have a title to the map and a map generating which is great. Hopefully some or all of this was helpful. Let me know if you want me to expand on anything or maybe another reviewer or a professor/ instructor will have more ideas or suggestions!

**Review by brglea — CHANGES_REQUESTED** [review]
@WillToon great work here, I left seperate comments on each file that will need to be addressed. 

Additionally I know this step was optional but reccomended - when you get your map to generate correctly, uploading it to your portfolio to show off what you can do! But this part will not be a part of the request changes, just wanted to mention it.

File: posts/will-toon/cu-map.md
Comment (WillToon) [reply posts/will-toon/cu-map.md:1]: Hey, @brglea thanks for bringing this to my attention! I am unsure how to change the file name once created as well as tweak the files (I have never used Github before so the navigation is confusing). Despite referring to that video the first time around something must have gone wrong when I first created the post.

I will be publishing a new pull request with the suggested edits and request you as an editor.

**Review by brglea — APPROVED** [review]
Awesome job @WillToon ! This looks fantastic! I see you changed the folder, you added citations and the map is now showing up with background and is a polygon. I forget how much vegetation/ how green Boulder looks, so cool!

@eculler can you take a look at this when you have a chance and approve/merge this pull request if everything looks good to you too? Thank you!

**Review by hanried — APPROVED** [review]
Both files, cu-map.md and cu.html, in the 01-first-map/will-toon folder look good! nicely done!

**Review by cmq879 — APPROVED** [review]

**Comment by eculler** [comment]
Thanks @WillToon I went ahead and merged!

**Pull request merged**
